stubs/google/cloud/ndb/__init__.pyi

```diff
@@ -1,4 +1,9 @@
-from .context import Context
+from .context import Context as Context
+from .query import (
+    AND as AND, OR as OR, Query as Query, Node as Node,
+    ConjunctionNode as ConjunctionNode, DisjunctionNode as DisjunctionNode,
+    FilterNode as FilterNode)
```

Comment:
Why is this written this way? (`smth as smth`)

Reply:
Because this is the only way to export the imported objects as part of the `__all__` list. If I don't do this, then MyPy fails because it can't find the names.

Reply:
Ok